read-the-docs-2017-stats.rst

```diff
@@ -0,0 +1,109 @@
+.. post:: Jan 26, 2018
+   :tags: stats, year-in-review
+
+Read the Docs 2017 Stats
+========================
+
+2017 has been a good year for Read the Docs.
+We've settled into a sustainability model that is working for us,
+and have started to grow our team to be able to better support the community.
+
+.. note:: 
+
+	You can always see our stats for the last `30 days`_. 
+
+	Our posts from 2013_, 2014_, 2015_, and 2016_ are also available.
+
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+.. _2016: https://blog.readthedocs.com/read-the-docs-2016-stats/
+
+Page Views
+----------
+
+Our stats:
+
+* 338 Million Page Views (up from 252M in 2016)
+* 75 Million Unique Visitors (up from 56M in 2016)
+
+.. From Google Analytics
+
+We remain one of the `largest sites`_ on the internet.
+Turns out *a lot of people* read documentation each day for open source projects!
+
+.. _largest sites: http://www.alexa.com/siteinfo/readthedocs.io
+
+Site Stats
+----------
+
+The stats, in total numbers:
+
+* 77,000 projects (up from 53k in 2016)
+* 103,000 users (up from 65k in 2016)
+
+We have been battling spam quite heavily this year,
+so these numbers might be a bit skewed.
+I believe they should be pretty accurate in terms of percentage growth though.
+
+.. Project.objects.count()
+.. User.objects.count()
+
+Community
+---------
+
+This year, we had:
+
+* 23 `people`_ who committed code to the main repository (up from 18 in 2016)
+* 1058 `commits`_ (similar to 1032 in 2016)
+* 513 `issues`_ - 194 open, 319 closed (up from 461 in 2016)
+
+.. https://github.com/rtfd/readthedocs.org/graphs/contributors?from=2017-01-01&to=2017-12-31&type=c
+.. git rev-list --count --all --max-age=1451606400 --min-age=1483228800
+.. is:issue  created:2017-01-01..2017-01-01 
+
+.. _people: https://github.com/rtfd/readthedocs.org/graphs/contributors?from=2017-01-01&to=2017-12-31&type=c
+.. _commits: https://github.com/rtfd/readthedocs.org/commits/master
+.. _issues: https://github.com/rtfd/readthedocs.org/search?utf8=%E2%9C%93&q=created%3A%3E%3D2017-01-01&type=Issues
+
+Funding
+-------
+
+* $250,000 in revenue (up from $75,000 in 2016)
+
+Read the Docs is sustainable.
```

Comment:
This point, left on its own like this, is presented matter-of-factly and doesn't really provide any supporting argument. This should either be drawn out to more of a statement or dropped altogether.

Reply:
Yep, will remove it.